src/Blocks/Magazine_Block.php

```diff
@@ -42,27 +43,37 @@ protected function get_key(): string {
 
 	protected function get_fields(): array {
 		return [
-			$this->get_title_field(),
-			$this->get_overline_field(),
+			$this->get_title_line_1_field(),
+			$this->get_title_line_2_field(),
```

Comment:
Could you please specify why do we need this extra title?

Reply:
TLDR; the design wants the block title to have two parts, each with a different size and font weight.

Reply:
👍